manifests/base/grafana/dash-k8s-compute-resources-pod.yaml

```diff
@@ -1914,8 +1916,8 @@ data:
             "allValue": null,
             "current": {
               "selected": false,
-              "text": "cert-manager-webhook-559cdf74cf-2hml5",
-              "value": "cert-manager-webhook-559cdf74cf-2hml5"
+              "text": "acm-custom-registry-d4f6cfc6d-f4trs",
```

Comment:
should we keep text/value as empty string in dashboard yaml?

Reply:
I think so yes - likely exists in all of the current dashboards - will update.